app/timetables/migrations/0001_initial.py

```diff
@@ -1,7 +1,8 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.9.7 on 2016-08-15 20:37
+# Generated by Django 1.9.7 on 2016-08-22 22:04
 from __future__ import unicode_literals
 
+import common.mixins
```

Comment:
Aunty Kosy, It's a generated file ma. The mixin was auto-included because of ForceCapitalizeMixin used in the model